niworkflows/func/util.py

```diff
@@ -403,16 +391,8 @@ def init_enhance_and_skullstrip_bold_wf(
     skullstrip_first_pass = pe.Node(
         fsl.BET(frac=0.2, mask=True), name="skullstrip_first_pass"
     )
-    bet_dilate = pe.Node(
-        fsl.DilateImage(
-            operation="max",
-            kernel_shape="sphere",
-            kernel_size=6.0,
-            internal_datatype="char",
-        ),
-        name="skullstrip_first_dilate",
-    )
-    bet_mask = pe.Node(fsl.ApplyMask(), name="skullstrip_first_mask")
+    bet_dilate = pe.Node(BinaryDilation(radius=6), name="skullstrip_first_dilate")
+    bet_mask = pe.Node(ApplyMask(), name="skullstrip_first_mask")
```

Comment:
we should probably take the chance to rename these, since we're changing the interfaces altogether